issue.go

```diff
@@ -245,3 +277,24 @@ func (c *client) DeleteIssue(id int) error {
 func (issue *Issue) GetTitle() string {
 	return issue.Tracker.Name + " #" + strconv.Itoa(issue.Id) + ": " + issue.Subject
 }
+
+func getIssueFilterClause(filter IssueFilter) string {
+	clause := ""
+	if filter.ProjectId != "" {
+		clause = clause + fmt.Sprintf("&project_id%v", filter.ProjectId)
```

Comment:
That is as I intended.
According to redmaine Rest-Api wiki, you could use ">" or "<" with these queries too, 
url : http://www.redmine.org/projects/redmine/wiki/Rest_Issues
it's simple solution.